Core/GDCore/Extensions/Builtin/AdvancedExtension.cpp

```diff
@@ -105,7 +105,8 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAdvancedExtension(
           "res/function32.png",
           "res/function16.png")
       .AddParameter("functionParameterName", "Parameter name")
-      .UseStandardRelationalOperatorParameters("number");
+      .UseStandardRelationalOperatorParameters(
+          "number", gd::ParameterOptions::MakeNewOptions());
```

Comment:
This should probably be the default parameter value.

Reply:
When talking about this with Alex, I was worried of default values in JS. In C++ I guess it could work, and I guess if we put the "optional" flag in Bindings.idl, Emscripten would generate a call with the default value (well, a call to the function without the parameter, so it would take the default value).

I think we should:
- Still keep the MakeNewOptions for safety for now and consistency.
- Add the default parameter so that we can then remove things to make them less verbose.